packages/webapp/src/_app/hooks/queries.ts

```diff
@@ -54,8 +55,16 @@ export const queryMemberGroupParticipants = (
 ) => ({
     queryKey: ["query_member_group_participants", memberAccount, config],
     queryFn: () => getMemberGroupParticipants(memberAccount, config),
+    enabled: Boolean(memberAccount),
 });
 
+export const queryVoteDataRow = (account: string | undefined) => ({
```

Comment:
Optional: use the undefined shorthand `(account?: string)` for these. But they do the same, as you know.

Reply:
and another great one. Thanks for pointing this out. This makes a lot of sense and tightens up plenty of the code... Thanks!